benchmark/Streamly/Benchmark/Prelude/Serial/Transformation.hs

```diff
@@ -148,11 +148,11 @@ tapRate n str = do
 {-# INLINE pollCounts #-}
 pollCounts :: Int -> SerialT IO Int -> IO ()
 pollCounts n =
-    composeN n (Internal.pollCounts (const True) f FL.drain)
+    composeN n (Internal.pollCounts (const True) f)
 
     where
 
-    f = Internal.rollingMap2 (-) . Internal.delayPost 1
+    f = S.drain . Internal.delayPost 1
```

Comment:
Don't remove rollingMap2, we need to do some processing on the stream to get a realistic code.